### PR TITLE
SKYR-1058: mongodb and prometheus configuration

### PR DESCRIPTION
**Pull request by PellePedro**

### CODE REVIEW

File: charts/worker/values.yaml [hunk 1]
```diff
@@ -58,4 +58,6 @@ bootConfig:
   config:
     managementPort: 35142
     backend: "kubernetes"
+    mongodbUrl: mongodb://root:secret@mongodb.skyramp:27017
```
Comment (kslee) [inline charts/worker/values.yaml:61]: can we use something different to username and password?

Comment (PellePedro) [reply charts/worker/values.yaml]: default credentials updated to admin/pa$$w0rd

File: charts/worker/values.yaml [hunk 2]
```diff
@@ -59,3 +59,14 @@ bootConfig:
     managementPort: 35142
     backend: "kubernetes"
   mountPath: "/etc/skyramp"
+
+ManagementPlaneConfig:
+  mongoConfig:
+    enabled: true
+    mongodbUrl: "mongodb://admin:pa$$w0rd@mongo-datastore-svc.skyramp:27017/?authSource=admin&authMechanism=SCRAM-SHA-256"
```
[the remark this answers is not in the document]
Comment (PellePedro) [reply charts/worker/values.yaml:66]: the configuration in the worker image, contains the default values maintaining a configuration would need to be done outside the image. We have the same case for the bootConfig where backend and managementPort is defined in the helm chart and as default constant in the docker image.
```
bootConfig:
  config:
    managementPort: 35142
    backend: "kubernetes"
  mountPath: "/etc/skyramp"
```

maybe @kslee has recommendations

File: charts/worker/Chart.yaml [hunk 3]
```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.16
```
[the remark this answers is not in the document]
Comment (PellePedro) [reply charts/worker/Chart.yaml:18]: yes you might be correct, I'll revert

[the remark this answers is not in the document]
Comment (texmachina) [reply charts/worker/Chart.yaml:18]: You have to manually +1 the version before merging to main for the updated chart. If the version already exists, the action will fail to publish the updated chart.

File: charts/worker/templates/_helpers.tpl [hunk 4]
```diff
@@ -63,6 +63,13 @@ Boot config
 {{- toYaml .Values.bootConfig.config }}
 {{- end }}
 
+{{/*
+Managet Plane config
```
Comment (sacheth003) [inline charts/worker/templates/_helpers.tpl:67]: Nit: `Management`

**Review by sacheth003 — APPROVED** [review]

File: charts/worker/templates/_helpers.tpl (hunk 4 above)
Comment (PellePedro) [reply charts/worker/templates/_helpers.tpl]: fixed

**Pull request merged**